app/codecs/CirceDecoders.scala

```diff
@@ -86,5 +87,15 @@ object CirceDecoders {
   implicit val switchStateDecode: Decoder[SwitchState] = deriveDecoder
   implicit val paymentMethodsSwitchCodec: Codec[PaymentMethodsSwitch] = deriveCodec
   implicit val switchesCodec: Codec[Switches] = deriveCodec
+  implicit val stripeApiErrorCodec: Codec[StripeApiError] = deriveCodec
```

Comment:
are we ever encoding a `StripeApiError` as JSON? if not then we only need an implicit `Decoder`